LICENSE

```diff
@@ -1,8 +1,8 @@
    Puppet - Automating Configuration Management.
 
-   Copyright (C) 2005-2015 Puppet Labs Inc
+   Copyright (C) 2005-2016 Puppet, Inc.
```

Comment:
I had guidance from Suzame Tong.  I don't know who in legal to contact about it.